src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -280,21 +280,31 @@ module Ledger = struct
     let%map () = Tar.create ~root:dirname ~file:tar_path ~directory:"." () in
     tar_path
 
-  let load ~genesis_dir ~logger ~constraint_constants
+  let load ~proof_level ~genesis_dir ~logger ~constraint_constants
       (config : Runtime_config.Ledger.t) =
     Monitor.try_with_join_or_error (fun () ->
         let add_genesis_winner_account accounts =
-          let pk, _ = Coda_state.Consensus_state_hooks.genesis_winner in
-          match accounts with
-          | (_, account) :: _
-            when Public_key.Compressed.equal (Account.public_key account) pk ->
-              accounts
-          | _ ->
-              ( None
-              , Account.create
-                  (Account_id.create pk Token_id.default)
-                  (Currency.Balance.of_int 1000) )
-              :: accounts
+          let add_genesis_winner_account =
```

Comment:
is it worth reprising the PR comment here, to say that we don't want to change the ledger layout for integration tests?